test/e2e/fixtures/preprod/get-protocol-parameters.e2e.ts

```diff
@@ -0,0 +1,70 @@
+import { expect } from 'vitest';
+
+export default [
+  {
+    testName: 'GET_PROTOCOL_PARAMETERS success - preprod',
+    result: {
+      data: {
+        epoch: expect.any(Number),
+        min_fee_a: 44,
+        min_fee_b: 155_381,
+        max_block_size: 90_112,
+        max_tx_size: 16_384,
+        max_block_header_size: 1100,
+        key_deposit: '2000000',
+        pool_deposit: '500000000',
+        e_max: 18,
+        n_opt: 500,
+        a0: 0.3,
+        rho: 0.003,
+        tau: 0.2,
+        decentralisation_param: 0,
+        extra_entropy: null,
+        protocol_major_ver: 9,
+        protocol_minor_ver: 0,
+        min_utxo: '4310',
+        min_pool_cost: '170000000',
+        nonce: '538e6395f3e5c8b415ac054e56ce4bfed41ce39c0fd2890c86d646a4a387b9b9',
```

Comment:
```suggestion
        nonce: expect.any(String),
```

this changes every epoch, same as epoch number

Reply:
c9ce170 should solve this